VotingApplication/VotingApplication.Web/Api/SignalR/ClientSignaller.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNet.SignalR;
+
+namespace VotingApplication.Web.Api.SignalR
+{
+    public static class ClientSignaller
+    {
+        private static IHubContext getHubContext()
```

Comment:
In general private methods should be below the methods that call them, and public methods should be first.